### ISSUE TITLE
Embedding existing HTML plots as panels via panel_url()

### ISSUE THREAD

**Issue by Errordude** [hafen/trelliscopejs#128]
Hi, is it possible to add existing HTML plots as panels?
Thank you and sorry for a stupid question I spent a lot of time trying to do that.

**Comment by hafen** [comment]
Yes you can do so either by using any of the R [htmlwidgets packages](htmlwidgets.org) or by pointing to existing html pages. Note that there is a newer version of Trelliscope that you should check out: https://trelliscope.org/trelliscope/articles/trelliscope.html
